imap_processing/cdf/config/imap_ultra_l1a_variable_attrs.yaml

```diff
@@ -0,0 +1,52 @@
+#  <=== Coordinates ===>
+# TODO: revisit this whole file
```

Comment:
Noted.

Reply:
Thank you, Laura! I created this issue which has more details about which files need further updates. https://github.com/IMAP-Science-Operations-Center/imap_processing/issues/716. Hope that helps